src/dashboard/app.py

```
@@ -5,80 +5,259 @@
+    st.header("데이터 및 모델 모니터링")
+    
+    if monitor_rep:
+        st.subheader("Data Drift 탐지 결과 (PSI / KS-test)")
+        
+        metrics = monitor_rep.get("metrics", {})
+        metrics_df = pd.DataFrame.from_dict(metrics, orient='index').reset_index()
+        metrics_df.columns = ["Feature", "PSI", "KS Statistic", "KS p-value"]
+        
+        # PSI 기준 컬러링
+        def color_psi(val):
+            if val > 0.2: return 'background-color: #ff4b4b; color: white' # High drift
```

Ruff (0.15.14)

[error] Multiple statements on one line (colon)

(E701)